best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
       and 'resources.limits.memory' per pod. If a namespace level request or limit is specified, 
       defaults will automatically be applied to each pod based on the 'LimitRange' configuration.
 spec:
-  validationFailureAction: audit
+  validationFailureAction: enforce
```

Comment:
any reason why this was changed to `enforce`? For most of the sample policies, we keep as `audit` and allow the user to set as enforce if they wish.

Reply:
Not any specific reason. I was testing using enforce. I will revert back to audit.